builtin/credential/cert/path_certs.go

```diff
@@ -124,17 +137,26 @@ func (b *backend) pathCertRead(
 		return nil, nil
 	}
 
-	duration := cert.TTL
-	if duration == 0 {
-		duration = b.System().DefaultLeaseTTL()
+	ttl := cert.TTL
+	if ttl == 0 {
+		ttl = b.System().DefaultLeaseTTL()
 	}
 
+	maxTTL := cert.MaxTTL
+	if maxTTL == 0 {
+		maxTTL = b.System().MaxLeaseTTL()
+	}
+
+	period := cert.Period
+
 	return &logical.Response{
 		Data: map[string]interface{}{
 			"certificate":  cert.Certificate,
 			"display_name": cert.DisplayName,
 			"policies":     cert.Policies,
-			"ttl":          duration / time.Second,
```

Comment:
I think we should actually not display ttl, max_ttl, or period if they're not set. This is because the value will vary according to the system/mount tuning, so returning 0 (or not returning the key at all) keeps people from making assumptions based on snapshots of the current values.

Reply:
Period will be 0 if that's not set. I can remove ttl and max_ttl from using the sys values if those are zero so that they return zero as well. AFAIK the other backends return ttl, max_ttl, and period all the time so if we do decide to remove them if unset we would have to do so for other backends as well, though this would could be a breaking change on people using these values from the response.

If this value will vary and we are returning 0 or hiding it if it's unset, where can the user/operator find out what these values are, would it be by doing a token lookup directly?